test/kind/image.go

```diff
@@ -57,6 +61,15 @@ func (k *Kind) LoadFakeAPIServer() error {
 	return k.loadImage(binPath, mainPath, image, dockerfilePath)
 }
 
+func (k *Kind) LoadAuditWebhook() error {
+	binPath := filepath.Join(k.rootPath, "./test/tools/audit-webhook/bin/audit-webhook")
+	dockerfilePath := filepath.Join(k.rootPath, "./test/tools/audit-webhook")
+	mainPath := filepath.Join(dockerfilePath, "cmd")
+	image := "audit-webhook-e2e"
```

Comment:
nit: can this refer to the exported `const` definition of this? I remember seeing it somewhere...